backend/core/src/units/entities/unit.entity.ts

```diff
@@ -127,11 +130,15 @@ class Unit {
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   sqFeet?: string | null
 
-  @Column({ nullable: true, type: "text" })
+  @Column({ 
+    type: "enum",
+    enum: UnitStatus,
+    default: UnitStatus.unknown,
+  })
   @Expose()
-  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
-  @IsString({ groups: [ValidationsGroupsEnum.default] })
-  status?: string | null
+  @IsEnum(UnitStatus, { groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty({ enum: UnitStatus, enumName: "UnitStatus" })
+  status: UnitStatus
```

Comment:
I guess I don't know why this doesn't change the UnitDto? It seems like you're making this a required field and an enum instead of a string. Shouldn't that also be reflected in the data object?

Reply:
Yeah sorry you're right, I do have to update backend-swagger. But it wasn't an issue, I saw the problems you ran into and they were easy to fix.